python/llm/example/GPU/HF-Transformers-AutoModels/Model/mixtral/README.md

```diff
@@ -24,12 +24,30 @@ pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-w
 pip install transformers==4.36.0
 ```
 
-### 2. Configures OneAPI environment variables
+### 2. Download Model and Replace File
+If you select the Mixtral model ([DiscoResearch/mixtral-7b-8expert](https://huggingface.co/DiscoResearch/mixtral-7b-8expert)), please note that `flash_attn` dependency is required. To address this issue, we have provided updated file ([DiscoResearch-mixtral-7b-8expert/modeling_moe_mistral.py](./DiscoResearch-mixtral-7b-8expert/modeling_moe_mistral.py)), which can be used to run on Intel GPUs.
```

Comment:
To run [DiscoResearch/mixtral-7b-8expert](https://huggingface.co/DiscoResearch/mixtral-7b-8expert) model on Intel GPU, we have provided an updated version of ([DiscoResearch-mixtral-7b-8expert/modeling_moe_mistral.py](./DiscoResearch-mixtral-7b-8expert/modeling_moe_mistral.py)).